src/perl/vdotest/vdotests.suites

```diff
@@ -811,6 +811,23 @@ $suiteNames{debugKernelTests}
      @{$suiteNames{checkin}},	# A bunch of basic VDO operation tests.
     ];
 
+###########################################################################
+# Some tests that exercise various kernel interface areas, to run on
+# mainlineNext.
+##
+$suiteNames{mainlineNextTests}
+  = [
+     @{$suiteNames{checkin}},	# A bunch of basic VDO operation tests.
+     "Create03",		# Test that many restarts works
+     "DiscardPerf",     	# Test that normal discards work. Longer.
+     "Discard512",		# Test that 512 byte discards work
+     "Dmsetup",			# Test that dmsetup interface works
+     "Full01",			# Test filling a device works. Longer.
+     "Logical512",		# Test that 512 byte emulation works
+     "Suspend",			# Test that suspending works
+     "VDOStats",		# Test that vdostats works
+    ];
+
 ###########################################################################
```

Comment:
Do we realistically expect that the tests for mainline-next will be different than the ones we want to run on rawhide? If not, we should just have one suite. (The name of the suite could be updated to be more generic, maybe.)

Reply:
OK.  Let me update it.